code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -356,7 +357,28 @@ GLOBAL_LIST_EMPTY(PDAs)
 					dat += "None detected.<br>"
 				else if(cartridge && cartridge.spam_enabled)
 					dat += "<a href='byond://?src=[REF(src)];choice=MessageAll'>Send To All</a>"
-
+			if(6)
+				dat += "<h4>[PDAIMG(mail)] ExperTrack® V4.26.2</h4>"
+				dat += "<i>Thank you for choosing ExperTrack® brand software! ExperTrack® inc. is proud to be a NanoTrasen employee expertise and effectiveness department subsidary!</i>"
+				dat += "<br><br>This software is designed to track and monitor your expertise as a NanoTrasen employee. Your performance in different areas of expertise has been categorized below.<br><HR>"
+				for (var/type in subtypesof(/datum/skill))
+					var/datum/skill/S = GetSkillRef(type)
+					var/skill_level_num = user.mind.get_skill_level(S)+1
+					var/skill_level_name = uppertext(SSskills.level_names[skill_level_num])
+					var/skill_exp = user.mind.skill_experience[S]
+					var/required_exp = user.mind.get_xp_to_level_up(S)
+					var/exp_percent = skill_exp/required_exp
+					var/exp_string = num2loadingbar(exp_percent, 10)
+					dat += "<br><b>[S.name]</b>"
+					dat += "<br><i>[S.desc]</i>"
+					dat += "<br>"
+					dat += "<ul><li>"
+					dat += "<br>CURRENT EMPLOYEE LEVEL OF EXPERTISE: [skill_level_name]"
+					if (required_exp != -1)

```

Comment:
combined with my other comment, this can just be if(required_exp)

Reply:
ur genius is showing